app/workers/Conveyor.scala

```diff
@@ -37,8 +44,34 @@ object Conveyor {
     val mapSubs = allSubs.groupBy(_.subscriberId)
     val actMap = mapSubs.map { case (subId, subs) => (subId, subs.groupBy(_.action)) }
     actMap.map { case (subId, subMap) => processSubs(item, subMap) }
-    //val channel = Channel.findById(sub.channel_id).get
-    //transferItem(transfer, item, channel, Some(topic))
+  }
+
+  def fulfill(sub: Subscription) = {
+    // transfer all eligible items ('latest' checking NYI)
+    sub.topic.get.itemsSince(sub.earliest).foreach(convey(_, sub))
+  }
+
+  def transferItem(item: Item, subscr: Subscriber) = {
+    // one-off (not subscription-based) transfer request from a subscriber
+    // presumed to be a delivery, but this could be generalized if needed
+    val subTopic = Topic.forSchemeAndTag("meta", "picks:" + subscr.id).getOrElse(
+      Topic.make(Scheme.findByTag("meta").get.id, "picks:" + subscr.id, subscr.name))
```

Comment:
Opened #116 to investigate whether this introduces unexpected consequences in other methods (overall I think it's great and we can tweak other methods to adjust to the new pseudo Topic).